src/visualizers/panels/ExecutionIndex/ExecutionIndexControl.js

```diff
@@ -96,12 +96,12 @@ define([
         let title = desc.plotlyData.layout.title ? desc.plotlyData.layout.title : {};
 
         if (title.text) {
-            title.text = `${title.text} (${desc.abbr})`;
+            title.text = `${title.text} (${desc.abbr || desc.jobName})`;
```

Comment:
Is `desc.abbr` ever falsey? I am actually not sure we need to abbreviations anymore since we aren't combining plot values. (Now, just adding the job/execution name to the title is probably more appropriate.)

Reply:
nevermind. It is falsey when only one execution is selected.